lib/cmgd_frntnd_client.c

```diff
@@ -915,7 +915,7 @@ cmgd_result_t cmgd_frntnd_create_client_session(
 	sessn->user_ctxt = user_ctxt;
 	sessn->client_id = client_id;
 	sessn->clnt_ctxt = clnt_ctxt;
-	sessn->session_id = 0;
+	sessn->session_id = &sessn;
```

Comment:
No on the frontend client side session id is assigned from server via a SESSION_REPLY message.. On receiving a SESSION_REPLY message the sessn->session_id is initialised with whatever value has been assigned by the server. This is NOT a bug.